services/invitations/README.md

```diff
@@ -7,11 +7,2789 @@ Users invited via this Invitation Manager (libre graph API) will have `userType=
 The corresponding CS3 API [user types](https://cs3org.github.io/cs3apis/#cs3.identity.user.v1beta1.UserType) used to reperesent this are: `USER_TYPE_GUEST` and `USER_TYPE_PRIMARY`.
 
 
+## Subcommand
+
+This service introduces the `invitations` subcommand to `ocis`.
+
 ## Provisioning backends
 
-When oCIS is used for user management the users are created using the `/graph/v1.0/users` endpoint. For larger deployments the keycloak admin API can be used to provision users. We might even make the endpoint, credentials and body configurable using templates.
+### Keycloak
+
+The default (and currently only) backend used to handle Invitations is [Keycloak](https://www.keycloak.org/).
+Keycloak is an open source identity and access management (IAM) system that other OCIS services also support as an
+authentication and authorization backend.
+
+
+#### Realm configuration
+
+This is the configuration of the Keycloak realm that this backend has been configured with, this includes
+the configuration of the client (invitations):
```

Comment:
```suggestion
This is the configuration of the Keycloak realm that this backend has been configured with and includes the client configuration for invitations:
```
Questions:
* to use this json, are there any prerequisites that need to be met on the KeyCloak side?
* the json is pretty large making the document hard to read.
I am thinking about to have that as linked file. Like in a folder named `md-sources` (can be re-used for images too).
This directory name is currently not used in any service.

Reply:
1. No.
2. That's not a bad idea.